src/persist-types/src/dyn_struct.rs

```diff
@@ -212,32 +214,31 @@ impl DynStructCol {
         })
     }
 
-    pub(crate) fn to_arrow_struct(&self) -> Option<(StructArray, Vec<Encoding>)> {
-        let (mut fields, mut encodings, mut arrays) = (Vec::new(), Vec::new(), Vec::new());
+    pub(crate) fn to_arrow_struct(&self) -> Option<StructArray> {
+        let (mut fields, mut arrays) = (Vec::new(), Vec::new());
         for (name, _stats_fn, col) in self.cols() {
-            let (encoding, array, is_nullable) = col.to_arrow();
+            let (array, is_nullable) = col.to_arrow();
             fields.push(Field::new(name, array.data_type().clone(), is_nullable));
-            encodings.push(encoding);
             arrays.push(array);
         }
         if fields.is_empty() {
             return None;
         }
-        let array = StructArray::new(ArrowLogicalType::Struct(fields), arrays, None);
-        Some((array, encodings))
+        // TODO(parkmycar): We need to pass the validity bitmap here.
```

Comment:
What's the difficulty here?

Reply:
Nothing in particular, I just wanted to make that change in a separate PR to prevent any possible behavior changes being introduced here

Reply:
Makes sense, thank you!